src/Bones/babylon.skeleton.ts

```diff
@@ -232,7 +232,10 @@
                     }
                 }
 
-                bone.getInvertedAbsoluteTransform().multiplyToArray(bone.getWorldMatrix(), targetMatrix, index * 16);
+                if (bone._index !== -1) {

```

Comment:
I think we should add an else with:
bone.getInvertedAbsoluteTransform().multiplyToArray(bone.getWorldMatrix(), targetMatrix, index * 16);